local_modules/stratum-pool/lib/jobManager.js

```diff
@@ -305,16 +301,30 @@ var JobManager = module.exports = function JobManager(options){
             port: port,
             worker: workerName,
             height: job.rpcData.height,
-            blockReward: job.rpcData.reward,
+            blockReward: getBlockReward(job.rpcData.height),
             difficulty: difficulty,
             shareDiff: shareDiff.toFixed(8),
             blockDiff: blockDiffAdjusted,
             blockDiffActual: job.difficulty,
-            blockHash:completeHeaderHash.toString('hex'),
+            blockHash: completeHeaderHash.toString('hex'),
             blockHashInvalid: blockHashInvalid
         }, blockHex, nTime, nonce, new Buffer(soln.slice(6), 'hex').toString('hex'), job.headerHash);
 
         return {result: true, error: null, blockHash: blockHash};
     };
+
+    let getBlockReward = function (blockNumber) {
+        const blockReward = 1500000000000000000;
+        const magnitude = 1000000000000000000;
+        const rampUpLowerBound = 0;
+        const rampUpUpperBound = 259200;
+        const delta = rampUpUpperBound - rampUpLowerBound;
```

Comment:
Given the constant values of `rampUpLowerBound` and `rampUpUpperBound` isn't this their delta always going to be the same? Why the need for 3 constants?

Reply:
This is taken from the kernel and I just wrote it for node. The readability is better this way than to use just the plain numbers for `delta`